### ISSUE TITLE
Date Picker Combobox Example - HTML source code example has wrong contrast ratio

### ISSUE THREAD

**Issue by giacomo-petri** [w3c/wai-aria-practices#264]
In [Date Picker Combobox Example](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-datepicker/) the HTML source code is not properly formatted, causing a color contrast issue for green text over light grey background, as per attachment.

<img width="863" alt="screenshot highlighting insufficient contrast ratio" src="https://github.com/w3c/wai-aria-practices/assets/106394951/720afe5a-da43-4331-90f4-44006a01c23e">

Opening a PR.

**Comment by giacomo-petri** [comment]
closing, I've just realised I was in wai-aria-practices, instead of aria-practices
